README.md

```diff
@@ -24,6 +29,19 @@ This branch currently represents the release branch.
 
 Experimental and WIP
 
+<<<<<<< HEAD
+# Update ros2
+To add or update part of ros2, for example 'rclc', branch 'foxy':
+1. Build ros2 https://docs.ros.org/en/foxy/Installation/Ubuntu-Development-Setup.html
+2. Clone part:
+git clone -b foxy https://github.com/ros2/rclc.git
+3. Make sure you installed colcon https://docs.ros.org/en/foxy/Tutorials/Colcon-Tutorial.html#create-a-workspace
+4. Build .so lib:
+. ~/ros2_foxy/install/local_setup.bash
+colcon build --packages-up-to --allow-overriding rclc
+(or without --allow-overriding, it's only in my case)
+5. Copy-replace includes and lib to appropriate folders to rclUE/Source/ThirdParty/ros2lib
```

Comment:
Seems to be renamed rclUE/Source/ThirdParty/ros now?

Reply:
Some issue was during merging, I update this file